src/category_theory/monad/adjunction.lean

```diff
@@ -33,6 +33,13 @@ end adjunction
 
 namespace monad
 
+/--
+Gven any adjunction `L ⊣ R`, there is a comparison functor `monad.comparison R`
+sending objects `Y : D` to Eilenberg-Moore algebras for `L ⋙ R` with underlying object `R.obj X`.
+
+We later show that this is full when `R` is full, faithful when `R` is faithful,
```

Comment:
Would you mind giving fully qualified declaration names?

Reply:
Ok, I've done this, but I'm not sure that it's helpful --- the name I'm referring to here is the current declaration.